File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.java
import java.util.Arrays;
import java.util.Scanner;
public class Main {
	

	public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    
    for(;;) {
    	String A=scan.nextLine();
    	if( A.equals("0")) {break;}
    	
    	char[] S=A.toCharArray();
    	int count =0;
    	for(int i=0;i<S.length;i++) {
    		if(S[i]==S[S.length-1-i])// 121  0배열==2배열 1배열==1배열 2배열==0배열
    			count++;
    	}
    	if(count==S.length) {
    		System.out.println("yes");
    	}else {System.out.println("no");}
      }
		
		
	}
}

File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.cc
#include <stdio.h>
#include <string.h>
#include <math.h>

int main() {
    char s[6];
    
    while (1) {
        scanf("%s", s);
        int x = strlen(s);
        int count = 0;
        if (x == 1 && s[0] == '0') { break; }
        for (int i = 0; i < x; i++) {///5--- s[0]-s[4] s[1]-s[3] s[2]-s[2] s[3]-s[1] s[4]-s[0] 4--s[0]-s[3] s[1]-s[2] s[2]-s[1]
            if (s[i] == s[x - 1 - i])
                count++;
        }if (count == x) { printf("yes\n"); }
        else { printf("no\n"); }
    }


  
        return 0;
}